### DISCUSSION TITLE
Part related issues

### DISCUSSION THREAD

**Discussion by likeproblem** [likeproblem/RestockRecolour#9] [Collaborator Disscusion]
VERY_COOL_PLACEHOLDER

**Comment by likeproblem** [comment]
Remember Shielded Docking Port? The issue was it had same name meshes with different textures, but I think I found fix for it! I just yoinked model, renamed one of the meshes, and viola! Will now properly color it ;)

**Reply by UncleMateo** [reply]
I'm having a similar problem with the Probodyne cores (aside from the Rovemate one, which I believe you called dibs on).  Stayputnik is 100% done, looks great in both versions, but the problem is the stock-grey and shiny-grey varieties use different textures, with their meshes for each being named the same.  I'll try some other weird stuff in the meanwhile, see if I can find an alternate workaround.. but could you elaborate on this process of model-yoinking and subsequent mesh-renaming? I believe that's the solution for the issue presently holding me up, given the multiple meshes with duplicate names.

![image](https://github.com/user-attachments/assets/55ab2fc1-9de4-4ff5-ba13-e3f44d728623)
here, I'll show you: recolor looks fine, but trying to switch from 'Paint' to 'Default' doesn't work for one of the variants.  It's just stuck in 'Paint'.

**Reply by likeproblem** [reply]
You can enumerate meshes like MODULES now, ex "mesh = CoolMesh,1", in latest TU update

**Reply by UncleMateo** [reply]
Oooh that's great!  Thanks!

**Comment by likeproblem** [comment]
Also anybody happens to have toolkit from here? http://forum.kerbalspaceprogram.com/index.php?/topic/160487-parttools-updated/

Oh boy, forums went down in most unfortunate time ever :(

**Comment by Leonardon9** [comment]
This part seems to have issues when lights on.
![Snipaste_2024-11-22_21-40-07](https://github.com/user-attachments/assets/777bc1b2-cd9d-48c9-a247-b31a8f4a5fad)
Also after adding this mod, pre-existing spacecrafts in my save has bugs on textures, like this-
![Snipaste_2024-11-22_21-44-52](https://github.com/user-attachments/assets/3e823d21-7bf4-4f3c-ae79-99024d9f8c5c)
in order to show normally, I have to swich back to VAB and swich to the spacecraft again。

**Reply by likeproblem** [reply]
1. Should be an easy fix
2. That should be expected, as you need to change scenes for craft to load up TU modules

**Reply by likeproblem** [reply]
also, what is the part in the picture? Can't really remember

**Reply by Leonardon9** [reply]
The Rockomax HubMax Multi-point Connector.

**Reply by UncleMateo** [reply]
Okay so I saw this last night, but couldn't immediately login to respond (I had been slacking off for so long GitHub was making me verify)

That is indeed the part I thought, 'stationHub' is its internal name.

My initial thought was, oh hey, that's gotta be the general/placeholder emissive texture being used in place of a part-specific emissive texture, in this case I think it would be the general endcap emissive used mostly by the command-type parts.. but the weird thing is, on my machine, I wasn't able to replicate that.  The original file just worked, lit up correctly.  I decided to try and update the config as if it _didn't_ work for me, and that still seems to work on my machine, so I'll push it as soon as I can, perhaps that will solve your issue.

Meanwhile, I noticed all the oscar-type mini fuel tanks seem to have their configs broken, I tried to fix that last night, to no avail.  They're giving that **Not Found** error.  I tried a few things, thought I had it for sure, but they didn't work.

**Reply by Leonardon9** [reply]
Thanks，and I also found out today that the whole FL-T(1.25m)type fuel tanks are all stuck on one theme.

**Comment by Leonardon9** [comment]
The Full FL-T Series(1.25m)fuel tank are stuck in one variant
![Snipaste_2024-11-28_19-24-26](https://github.com/user-attachments/assets/3a10c451-fefc-42fb-b124-5994793cc013)
